content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,181 @@
+linkTitle: Stability Proposal Announcement # Mandatory, make sure that your short title.
+date: 2025-10-23 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Austin Parker](https://github.com/austinlparker) (honeycomb.io)
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+body_class: otel-with-contributions-from # TODO: remove this line if there are no secondary contributing authors
+issue: the issue ID for this blog post # TODO: See https://opentelemetry.io/docs/contributing/blog/ for details
+sig: Governance Committee # TODO: add the name of the SIG that sponsors this blog post
+---
+
+<!-- If your post doesn't have secondary authors, then delete the following paragraph: -->
+
+With contributions from, and on behalf of, the OpenTelemetry Governance
+Committee.
+
+OpenTelemetry is a large, complex, and highly involved project. A quick glance
+at the
+[specification compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md)
+should be enough to tell you that supporting tracing, metrics, logs, and
+profiles across a dozen languages – not to mention the complexity that comes
+from different language runtimes and execution environments – means that there’s
+a _lot_ to take in. One thing we’ve heard from you over the past months is that
+this complexity makes it difficult to get started, or get going, with
+OpenTelemetry as an end user. It also makes life difficult for our maintainers
+and contributors. How should they prioritize feature development versus spec
+changes versus instrumentation stability?
+
+With this in mind, the OpenTelemetry project would like to announce some
+proposed changes that we’d like to institute, with the goal of normalizing our
+language around component stability and component offerings.
+
+We believe that these changes need community involvement and discussion to be a
+success, so we’re taking this opportunity to announce our intention and open an
+[issue for discussion](https://github.com/open-telemetry/community/issues/3086)
+in order to get feedback from users, maintainers, and contributors. We do not
+anticipate that these changes will be completed overnight, and want to assure
+everyone that we will continue to prioritize our existing commitments to users
+and maintainers even as we consider necessary changes for the overall wellbeing
+and maturity of the project.
+
+## Improving defaults for stable components
+
+The first proposal has to do with how SIGs and the project communicate stability
+of components – such as instrumentation libraries, Collector receivers, or tools
+like weaver. We will clearly state on the project website which components are
```

Comment:
do you mean, we will add stability information to the OpenTelemetry registry, and allow people to filter by stability?